### ISSUE TITLE
when i load a bninception model in tsn？ some wrong happen. 

### ISSUE THREAD

**Issue by poppingcode** [Cadene/pretrained-models.pytorch#55]
i load a bninception model---https://github.com/Cadene/pretrained-models.pytorch/tree/master/pretrainedmodels/models
It has been pre trained.

when i load a bninception model in tsn---https://github.com/yjxiong/tsn-pytorch/
After dozens of epoch, loss has been stuck in a large number.
i don't know how to solve it ,please help me ,or Or put forward a bit of thought。
thanks！！！！！

**Comment by Zhuysheng** [comment]
@poppingcode  do you solve it

**Comment by Cadene** [comment]
Did you try a momentum=0.1 ?
